test/mongoose_domain_core_SUITE.erl

```diff
@@ -2,50 +2,123 @@
 
 -compile(export_all).
 
+-include_lib("eunit/include/eunit.hrl").
+
+-define(STATIC_PAIRS, [{<<"example.cfg">>, <<"type #1">>},
+                       {<<"erlang-solutions.com">>, <<"type #2">>},
+                       {<<"erlang-solutions.local">>, <<"static type">>}, %% not allowed type
+                       {<<"example.org">>, <<"type #2">>}]).
+-define(ALLOWED_TYPES, [<<"type #1">>, <<"type #2">>, <<"type #3">>]).
```

Comment:
Interesting, so we are testing the cases that cannot be configured (static pairs have to be `{Host, Host}`) here. Did you want explicitly that?

Reply:
While it always will be `{Host, Host}`, I just didn't want to have this limit to be hardcoded in the module.